lib/git/branch_shortcuts.sh

```diff
@@ -32,14 +32,12 @@ EOF
 )"
 
   # Set numbered file shortcut in variable
-  local e=1
-  IFS=$'\n'
+  local e=1 IFS=$'\n'
```

Comment:
I’m partial to keep things on separate lines, though declaring using `local` is definitely the right move.